### PR TITLE
Update for latest exoservice and exo-run

### PR DESCRIPTION
**Pull request by trushton**
@kevgo @hugobho @martinjaime 
Updates the SpaceTweet app to work with the published version of exorelay (0.19.5) and the version of exo-run that is currently awaiting approval.

### CODE REVIEW

File: web-server/.dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (kevgo) [inline web-server/.dockerignore]: Why do we ignore this directory in Docker? It should be inside the docker image, no?

File: web-server/service.yml [hunk 2]
```diff
@@ -1,9 +1,10 @@
-name: SpaceTweet web server
+title: SpaceTweet web server
 description: Serves the web UI of SpaceTweet
+author: originate
 
 setup: npm install
 startup:
-  command: node_modules/.bin/lsc app --name {{name}} --exorelay-port {{exorelay-port}} --exocom-port {{exocom-port}}
+  command: "node_modules/.bin/lsc app"
```
Comment (kevgo) [inline web-server/service.yml]: Remove the double quotes

File: web-server/service.yml [hunk 3]
```diff
@@ -27,3 +28,10 @@ messages:
     - tweets.listed
     - tweets.created
     - tweets.deleted
+
+
+docker:
+  link:
+    - 'mongo'
```
Comment (kevgo) [inline web-server/service.yml]: @hugobho always writes `mongo:mongo` here

File: web-server/.dockerignore (hunk 1 above)
Comment (trushton) [inline web-server/.dockerignore]: It should be inside of the Docker image, but we want the node_modules to be **installed** inside of the Docker image for whatever system is being used as opposed to copying the local ones in. For example, if the docker image is running a linux distribution and I copy my osx node_modules into it, some packages may not work correctly.

Comment (kevgo) [inline web-server/.dockerignore]: You are totally right, one point for you!

**Comment by trushton** [comment]
All comments addressed.

**Pull request closed without merging**